Localizations/nl.lproj/Localizable.strings

```diff
@@ -353,18 +353,18 @@
 "Sets climate control temperature" = "Stelt de temperatuur van de Airconditioning in";
 "Show data timer on compatible complications" = "Toon datatimer op compatibele complicaties"; //Machine translated
 "Need help?" = "Hulp nodig?"; //Machine translated
-"Please login on your phone. Your login will automatically transfer to the watch." = "Gelieve in te loggen op uw telefoon. Uw login wordt automatisch overgedragen naar het horloge."; //Machine translated
+"Please login on your phone. Your login will automatically transfer to the watch." = "Gelieve in te loggen op je telefoon. Jouw login wordt automatisch overgedragen naar het horloge.";
 "Please install the iOS companion app, then login on your phone. Your login will automatically transfer to the watch." = "Installeer de bijbehorende iOS-app en log vervolgens in op uw telefoon. Uw login wordt automatisch overgedragen naar het horloge."; //Machine translated
 "Advanced" = "Geavanceerd"; //Machine translated
 "Inside / outside temp" = "Binnen / buiten temperatuur";
 "Are you sure? You will not be able to close your windows remotely" = "Weet je het zeker? Je kunt de ramen niet op afstand sluiten";
-"Never confirm complication actions, even irreversible ones and actions that can accidentally grant others access to your vehicle" = "Bevestig nooit complicatie acties, zelfs onomkeerbare acties en acties die anderen per ongeluk toegang tot uw voertuig kunnen geven";
-"Never confirm widget actions, even irreversible ones and actions that can accidentally grant others access to your vehicle" = "Bevestig nooit widgetacties, zelfs niet onomkeerbare acties en acties die anderen per ongeluk toegang tot uw voertuig kunnen geven"; //Machine translated
+"Never confirm complication actions, even irreversible ones and actions that can accidentally grant others access to your vehicle" = "Bevestig nooit complicatie acties, zelfs onomkeerbare acties en acties die anderen per ongeluk toegang tot je auto kunnen geven";
+"Never confirm widget actions, even irreversible ones and actions that can accidentally grant others access to your vehicle" = "Bevestig nooit widgetacties, zelfs niet onomkeerbare acties en acties die anderen per ongeluk toegang tot je auto kunnen geven"; //Machine translated
 "Wait For Wake" = "Wacht op ontwaken";
-"Vehicle not online, keep trying?" = "Voertuig niet online, blijven proberen?"; //Machine translated
+"Vehicle not online, keep trying?" = "Auto niet online, blijven proberen?";
 "Is Driving" = "Is aan het rijden"; //Machine translated
 "Show advanced" = "Toon geavanceerd"; //Machine translated
 "Hide advanced" = "Geavanceerd verbergen"; //Machine translated
-"Calculate charger power (kW) without neutral connection" = "Bereken het laadvermogen (kW) zonder nulleider"; //Machine translated
+"Calculate charger power (kW) without neutral connection" = "Bereken het laadvermogen (kW) zonder nulleider"; //Machine translated but needs clarification
```

Comment:
This is for an advanced toggle in settings where users can override how the charge power is calculated when the car is charging. Normally you would connect the tesla charger to neutral - but tesla allows the charger to be connected in a non-standard non-compatible fashion without a neutral connection. In this case, the charger power is less, and is calculated differently. There is no way to detect this weird connection in the api response, so had to be a toggle in settings.

Reply:
You can do the merges if you wish. If you change/fix the litle rendering issues I'll change it back. Now 2 words are hyphenated by one character and that's a bit awkward.